### PR TITLE
Dl 4278

### PR DESCRIPTION
**Pull request by jgrath**
# JIRA Story Dl 4278

*Reviewee* (Replace with your name)
 - [X]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [X]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)

*Reviewer* (Replace with your name)
 - [ ]  I've confirmed that every effort has been made to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [ ]  I've confirmed appropriate tests has been included with any code added (Unit, Integration, Acceptance etc.)
 - [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions
 - [ ]  I've confirmed code was added using logical, atomic commits, squashing as appropriate - including the JIRA issue number in the commit message
 - [ ]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/views/ConfirmationView.scala.html [hunk 1]
```diff
@@ -38,10 +38,14 @@
         title = Text(messages("confirmation.heading")),
         content = HtmlContent(messages("confirmation.text"))
     ))
-    <p class="govuk-body">@messages("confirmation.text.footer")</p>
 
+    @if(!paperLessAvailable) {
+        <p class="govuk-body">@messages("confirmation.text.footer", "in the post")</p>
```
Comment (grahampaulcook) [inline app/views/ConfirmationView.scala.html:43]: That hardcoded string "in the post" won't turn into Welsh should they toggle

File: app/views/ConfirmationView.scala.html [hunk 2]
```diff
@@ -38,10 +38,14 @@
...
+    } else {
+        <p class="govuk-body">@messages("confirmation.text.footer", "by email")</p>
```
Comment (grahampaulcook) [inline app/views/ConfirmationView.scala.html:45]: That hardcoded string "by email" won't turn into Welsh should they toggle

File: app/config/FrontendAppConfig.scala [hunk 3]
```diff
@@ -23,12 +23,16 @@ import play.api.i18n.Lang
...
 
 @Singleton
-class FrontendAppConfig @Inject() (configuration: Configuration) {
+class FrontendAppConfig @Inject()(configuration: Configuration) {
 
   val contactFormServiceIdentifier = "EEWFH"
 
+  val platformHost: Option[String] = configuration.getOptional[String]("platform.frontend.host")
+
+  val preferencesFrontendHost: String =
```
Comment (grahampaulcook) [inline app/config/FrontendAppConfig.scala:32]: I think this was fine before 
`val preferencesFrontendHost: String = configuration.get[Service]("microservice.services.preferences-frontend").baseUrl` as that would of picked up the value `preferences-frontend.public.mdtp` from app-config-base which would be fine in a connector call

Comment (grahampaulcook) [reply app/config/FrontendAppConfig.scala:32]: I have a feeling this was meant for **pertaxFrontendHost** instead of **preferencesFrontendHost**

Comment (jgrath) [reply app/config/FrontendAppConfig.scala]: Have a look now

File: app/controllers/ConfirmationController.scala [hunk 4]
```diff
@@ -43,7 +43,7 @@ class ConfirmationController @Inject()(
                                         getData: DataRetrievalAction,
                                         requireData: DataRequiredAction,
                                         val controllerComponents: MessagesControllerComponents,
-                                        val paperlessPreferenceConnector: PaperlessPreferenceConnector,
+                                        val paperlessPreferenceConnector: PaperlessPreferenceConnectorImpl,
```
Comment (grahampaulcook) [inline app/controllers/ConfirmationController.scala:46]: This needs to be `PaperlessPreferenceConnector` not the implementation class

Comment (jgrath) [reply app/controllers/ConfirmationController.scala]: Done

**Review by grahampaulcook — APPROVED** [review]

**Pull request merged**